java/http/java-http-consumer/src/main/java/HttpConsumer.java

```diff
@@ -108,6 +137,22 @@ public void createConsumer() throws IOException, InterruptedException, URISyntax
         }
     }
 
+    public void processProperty(Properties props, String k, Map<String, Object> map) {
+        if (props.stringPropertyNames().contains(k)) {
+            String v = props.getProperty(k);
+            Object obj;
+            if (isStringBoolean(v)) {
+                obj = Boolean.parseBoolean(v);
+            } else if (isStringNumber(v)) {
+                obj = Integer.parseInt(v);
+            } else {
+                obj = v;
+            }
+            map.put(k, obj);
+            log.info("key: {}, value: {}", k, v);
```

Comment:
I would remove this line since they will already be printed out with consumerInfo print line and it doesn't print the "name" and "format" configurations

Reply:
removed